src/jsonEditor.scss

```diff
@@ -1,3 +1,6 @@
+@import 'leaflet-draw/dist/leaflet.draw.css';
+@import 'leaflet/dist/leaflet.css';
```

Comment:
why are these imports in an unrelated scss file? Can't you just import them close to where they're used (https://github.com/open-formulieren/formio-builder/blob/main/src/registry/map/index.ts)? And preferably a `preview.scss` file that loads these libraries, so we have a consistent pattern of loading stylesheets for a given context: https://github.com/open-formulieren/formio-builder/blob/2d64b61d74ef79145a401dbe22debec0adef36cc/src/registry/email/preview.tsx#L7